content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Mastercontroller Cheat Sheet"
+linkTitle: "Cheat Sheet Mastercontroller"
+
+weight: 800
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available commands below.
+---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands) by executing all commands automatically.
+
+### Example
+
+- For displaying all the available command options you can run
+```powershell
```

Comment:
Do we plan to have Linux examples later too?

Reply:
Linux examples are kinda "standard" for the doc, however we could put Linux and Windows examples if you think that's helpful.